### DISCUSSION TITLE
IPFX map does not match color with the reference.

### DISCUSSION THREAD

**Discussion by rumman06** [mtex-toolbox/mtex#627] [Ask Anything]
Dear MTEX community,

I am new to MTEX and also not much proficient with MatLab.  I am having an issue with reproducing the IPF maps in their proper orientation. The maps plotted with MTEX does not have the same color for the grains as the one I am getting from the Bruker ESpirit2. 

For example, please check the attached figures. The first one is from the bruker. I am getting the second one from MTEX. However , with another tool ATEX, I can match the colors with the Bruker file as seen in the 3rd one.

![image](https://user-images.githubusercontent.com/59967431/101932487-77769b80-3ba0-11eb-860f-c26b98f09985.png)

![image](https://user-images.githubusercontent.com/59967431/101932522-82313080-3ba0-11eb-946c-a71de7684c38.png)

![image](https://user-images.githubusercontent.com/59967431/101932548-8cebc580-3ba0-11eb-8b56-e3fc1963d631.png)

I  guess I have to change the plane to match the color.  Kindly help me with how to change the miller index of the plane while plotting the IPF maps.

Sincerely,
Rumman.

**Comment by kilir** [comment]
Hi Rumman,
to produce two IPF maps with identical colors, which on top are correct, you need to 
a) use the same colorkey
b) have the same definition of orientations (coordinate system, ref. system definition etc.)
To be able to help you need to be more detailed.
c) use the same specimen reference directions.

How is your orientation data defined in your Bruker machine?
How is your specimen coordinate system defined?
How is your crystal coordinate system defined? 
How did you import your data into mtex?
What colorcoding do you use in mtex, which one in your Bruker system?

Cheers,
Rüdiger
